src/scripts/main.js

```diff
@@ -357,4 +357,27 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an element with the class 'dashboard' in your HTML. If this element does not exist, the script will throw an error when trying to append rows to it.